### DISCUSSION TITLE
Does this work for warzone 2 yet?

### DISCUSSION THREAD

**Discussion by dawright83** [TodoLodo2089/cod-python-api#18] [Q&A]
Here's what I'm getting when trying to pull data for warzone 2?

Do I have an error or does warzone 2 not work?

![image](https://user-images.githubusercontent.com/78715476/204935225-062e8c47-8e0a-4cad-ba98-95246a98340c.png)
![image](https://user-images.githubusercontent.com/78715476/204935278-52f68b34-8522-4e46-8005-b466280b4a6d.png)

**Comment by thedavidhanks** [comment]
I see the same result for a Battlenet platform

**Comment by FkngSer** [comment]
Yep it is still throwing these errors :(
